core/src/main/java/hudson/cli/CreateViewCommand.java

```diff
@@ -50,7 +54,27 @@ public String getShortDescription() {
     protected int run() throws Exception {
 
         final Jenkins jenkins = Jenkins.get();
-        jenkins.checkPermission(View.CREATE);
+
+        ModifiableViewGroup vg = jenkins;
+        if (viewName != null) {
+            int i = viewName.lastIndexOf('/');
+            if (i > 0) {
+                String group = viewName.substring(0, i);
+                Item item = jenkins.getItemByFullName(group);
+                if (item == null) {
+                    throw new IllegalArgumentException("Unknown ItemGroup " + group);
```

Comment:
We should not expose class names to users. Plus, we're not looking for an `ItemGroup` here.

```suggestion
                    throw new IllegalArgumentException("Unknown item group " + group);
```
or perhaps
```suggestion
                    throw new IllegalArgumentException("Unknown folder " + group);
```
or
```suggestion
                    throw new IllegalArgumentException("Unknown item " + group);
```
because at this point we haven't even checked whether it's a `ViewGroup`.